### ISSUE TITLE
Enh: try with NN models

### ISSUE THREAD

**Issue by jmzhao** [dolremi/Titanic-kaggle#1]
I am going to apply some multi-layer neural networks on some most significant features to see how it works out, with no or minimal feature engineering.

**Comment by jmzhao** [comment]
Start with features ['Sex', 'Age', 'Age_invalid', 'Pclass', 'Fare_invalid', 'LogFare'].
Got training accuracy 0.81 after 50 eppch.
Model is MLP with 3 hidden layers:
Input(6), Dense(16, tanh), Dense(32, tanh), Dropout(0.25), Dense(32, tanh), Dropout(0.5), Dense(1, sigmod)
